spyder/app/mainwindow.py

```diff
@@ -246,6 +247,28 @@ def setup_logging(cli_options):
                             filename=log_file,
                             filemode='w+')
 
+
+def qt_message_handler(msg_type, msg_log_context, msg_string):
+    """
+    Qt warning messages are intercepted by this handler.
+
+    On some oprating systems, warning messages might be displayed
+    even if the actual message does not apply. This filter adds a
+    blacklist for messages that are being printed for no apparent
+    reason. Anything else will get printed in the internal console.
+
+    On DEV mode, all messages are printed.
+    """
+    BLACKLIST = [
+        'QMainWidget::resizeDocks: all sizes need to be larger than 0',
+    ]
+    if DEV or msg_string not in BLACKLIST:
+        print(msg_string)  # spyder: test-skip

```

Comment:
@ccordoba12 do we want this printed to the internal console or we want this on the terminal?

If so how is that accomplished?

Thanks

Reply:
If you want to print to the terminal, you need to use the `logging` module. It can be `logger.info` to not miss the message among the huge amount produced by `logger.debug`.

I mean, I say it because you can start Spyder like this

    python bootstrap.py -- --debug-info minimal

to only see info messages.

Reply:
I think `print` does what I had more in mind for `dev` mode

Reply:
Ok, good to know :+1: